tools/cdb2sql/cdb2sql.c

```diff
@@ -694,7 +695,9 @@ static void process_line(char *sql, int ntypes, int *types)
     sqlstr[len] = '\0';
 
     int start_time_ms, run_time_ms;
+    gbl_in_stmt = 1;
```

Comment:
Use of gbl_in_stmt seems redundant to me. If we have a nonce, let's send it to the server and let the server take care of terminating the query if a match found.